src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -229,22 +226,22 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
 
   const { ownedVenues } = useOwnedVenues({});
 
-  const backButtonOptionList =
-    ownedVenues.reduce(
-      (
-        obj: Record<string, WithId<AnyVenue>> | null,
-        venue: WithId<AnyVenue>
-      ) => {
-        if (portal?.worldId !== venue.worldId || venue.id === portalId) {
-          return obj;
-        }
-
-        return { [venue.id]: venue, ...obj };
-      },
-      {}
-    ) ?? ALWAYS_EMPTY_OBJECT;
+  const backButtonOptionList = useMemo(
+    () =>
+      Object.fromEntries(
+        ownedVenues
+          .filter(({ id, worldId }) =>
+            portal?.worldId !== worldId || id === portalId ? false : true
```

Comment:
ternary operator can be removed by negating the condition, also De Morgan's law can be applied

Reply:
done 97a5880